src/service.c

```diff
@@ -161,6 +225,8 @@ static void r_on_name_acquired(GDBusConnection *connection,
 	if (r_context()->config->autoinstall_path)
 		auto_install(r_context()->config->autoinstall_path);
 
+	set_slot_status();
+
```

Comment:
This should be done after loading the config, so that the slot information is always available.